packages/zowe-explorer/src/job/init.ts

```diff
@@ -63,6 +63,18 @@ export async function initJobsProvider(context: vscode.ExtensionContext): Promis
             jobActions.refreshJob(node.mParent.mParent, jobsProvider);
         })
     );
+    context.subscriptions.push(
+        vscode.commands.registerCommand("zowe.jobs.downloadSingleSpool", async (node, nodeList) => {
+            const selectedNodes = getSelectedNodeList(node, nodeList) as IZoweJobTreeNode[];
+            await jobActions.downloadSingleSpool(selectedNodes, false);
+        })
+    );
+    context.subscriptions.push(
+        vscode.commands.registerCommand("zowe.jobs.downloadSingleSpoolBinary", async (node, nodeList) => {
+            const selectedNodes = getSelectedNodeList(node, nodeList) as IZoweJobTreeNode[];
+            await jobActions.downloadSingleSpool(selectedNodes, true);
+        })
+    );
```

Comment:
To avoid code duplication here, the handler could be adjusted based on a function parameter:

```ts
const downloadSpoolHandler = (isBinary: boolean) => async (node, nodeList) => {
    const selectedNodes = getSelectedNodeList(node, nodeList) as IZoweJobTreeNode[];
    await jobActions.downloadSingleSpool(selectedNodes, isBinary);
});
vscode.commands.registerCommand("zowe.jobs.downloadSingleSpool", downloadSpoolHandler(false));
vscode.commands.registerCommand("zowe.jobs.downloadSingleSpoolBinary", downloadSpoolHandler(true));
```